### ISSUE TITLE
unsupported format '.tgz'

### ISSUE THREAD

**Issue by askz** [mholt/archiver#111]
Hi, 
I get this when trying to extract a .tgz (which is a tar.gz)
```
⇒  arc ls ~/Downloads/srv_scripts.tgz                 
unsupported format '.tgz'
⇒  tar -xzvf ~/Downloads/srv_scripts.tgz 
scripts/
scripts/stop.sh
```

**Comment by mholt** [comment]
Fixed in 526228db02717285609233ec22606d727e53fd9c

**Comment by askz** [comment]
nice! that was fast thanks !
